src/js/handlebars.helpers.js

```diff
@@ -48,16 +46,59 @@ export default function handlebarsHelpers(Handlebars) {
       return options.inverse(this);
     }
   });
-  // Checks is expected type
-  Handlebars.registerHelper('isType', function (value, expected, options) {
+  // isType - Checks is expected type
+  handlebars.registerHelper('isType', function (value, expected, options) {
     if (value === expected) {
       return options.fn(this); // Render the block if condition is true
     } else {
       return options.inverse(this); // Render the else block if present
     }
   });
+  // ifAny - {{{#ifAny variable1 variable2 variable3 variable4 etc}}, if any set then return true
+  handlebars.registerHelper('ifAny', function (...args) {
+    const options = args.pop(); // The last argument is the options object
+    return args.some(arg => !!arg) ? options.fn(this) : options.inverse(this);
+  });
+  // formatDate - Format Date, for footer meta data i.e {{formatDate '2023-06-23'}}
+  handlebars.registerHelper('formatDate', function(dateString, defaultDate) {
+    // Use the dateString if provided, otherwise use the defaultDate, otherwise error
+    let date;
+    if (dateString) {
+      date = new Date(dateString);
+    }
+    if (isNaN(date) && defaultDate) {
+      date = new Date(defaultDate);
+    }
+
+    // Check if the date is valid
+    if (isNaN(date)) {
+      return 'Invalid Date';
+    }
+
+    var monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+    var day = date.getDate();
+    var month = monthNames[date.getMonth()];
+    var year = date.getFullYear();
+    return `${day} ${month} ${year}`;
+  });
+
+  // formatDateOrToday - Format Date if passed or today i.e. {{formatDateOrToday '2023-06-23'}} or {{formatDateOrToday}} <!-- This will use the current date -->
+  handlebars.registerHelper('formatDateOrToday', function(dateString) {
+    // Use the current date if dateString is missing or invalid
+    const dateToFormat = dateString || new Date().toISOString();
+    // Call the formatDate helper with the determined date and format
+    return handlebars.helpers.formatDate(dateToFormat);
+  });
 }
 
+//Only load once if Handlebars is available
 if(typeof(Handlebars) !== 'undefined') {
-  handlebarsHelpers(Handlebars);
+  this.registedHandlebarsHelpers = undefined;
```

Comment:
@duttonw could you have a look this one - "registed" might need to be "registrered".  

We're getting a JS error popup across Squiz

![image](https://github.com/qld-gov-au/qgds-bootstrap5/assets/126438309/799394fe-e95f-4395-aedf-d5b1b067af7e)

Reply:
should be fixed with https://github.com/qld-gov-au/qgds-bootstrap5/pull/299